src/main/java/io/reactivesw/category/infrastructure/validator/CategoryNameValidator.java

```diff
@@ -41,12 +41,17 @@ public static void validateNull(CategoryDraft draft) {
       LOG.debug("CategoryView name can not be null");
       throw new ParametersException("CategoryView name can not be null");
     }
+    if (draft.getName().getLocalized().entrySet().parallelStream()
+        .anyMatch(entry -> StringUtils.isEmpty(entry.getKey()) || StringUtils.isEmpty(entry.getValue()))) {
+      LOG.debug("CategoryView name can not be empty");
+      throw new ParametersException("CategoryView name can not be empty");
+    }
+
   }
```

Comment:
1. use stream instead of parallelStream.
2. use predicate.
3. log should end with a dot(.).